### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by victorShiva**
i have created in this file inside hide and show file

### CODE REVIEW

File: feature2/hide.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (shiva5775) [inline feature2/hide.html]: change hide file title

File: feature2/show.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (shiva5775) [inline feature2/show.html]: change show file title

File: feature2/show.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+    <h1>this is show.html file in feature2 branch</h1>
```
Comment (shiva5775) [inline feature2/show.html:9]: add some lorem files

**Review by shiva5775 — CHANGES_REQUESTED** [review]

File: feature2/hide.html (hunk 1 above)
Comment (victorShiva) [reply feature2/hide.html]: i have changed hide file title

File: feature2/show.html (hunk 2 above)
Comment (victorShiva) [reply feature2/show.html]: i have chenged show file title

File: feature2/show.html (hunk 3 above)
Comment (victorShiva) [reply feature2/show.html:9]: i have added lorem para

**Review by shiva5775 — APPROVED** [review]
well done victor

**Pull request merged**